src/page/HomePageTitle.tsx

```diff
@@ -0,0 +1 @@
+export const HomePageTitle = () => <h1 className="title">Home Page</h1>;
```

Comment:
i see u have 3 similar components with same styles where only the header changes.
Make 1 component that will accept the title as props